content/docs/hooks-overview.md

```diff
@@ -1,107 +1,107 @@
 ---
 id: hooks-overview
-title: Hooks at a Glance
+title: Seklias tentang Hooks
 permalink: docs/hooks-overview.html
 next: hooks-state.html
 prev: hooks-intro.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+*Hooks* merupakan penambahan baru pada React 16.8. *Hooks* dapat menggunakan *state* dan fitur React lainnya tanpa membuat sebuah kelas.
 
-Hooks are [backwards-compatible](/docs/hooks-intro.html#no-breaking-changes). This page provides an overview of Hooks for experienced React users. This is a fast-paced overview. If you get confused, look for a yellow box like this:
+*Hooks* merupakan fitur yang [*backwards-compatible*](/docs/hooks-intro.html#no-breaking-changes). Laman ini menyediakan ikhtisar mengenai *Hooks* untuk pengguna React yang berpengalaman. Laman ini merupakan ikhtisar yang serba cepat. Jika Anda kurang paham, carilah kotak kuning seperti di bawah ini:
 
->Detailed Explanation
+>Penjelasan Detail
 >
->Read the [Motivation](/docs/hooks-intro.html#motivation) to learn why we're introducing Hooks to React.
+>Baca laman [Motivasi](/docs/hooks-intro.html#motivation) untuk mengetahui kenapa kami mengenalkan *Hooks* pada React.
 
-**↑↑↑ Each section ends with a yellow box like this.** They link to detailed explanations.
+**↑↑↑ Setiap bagian diakhiri dengan kotak kuning seperti ini.** Kotak tersebut menghubungkan ke penjelasan yang lebih terperinci.
 
-## 📌 State Hook {#state-hook}
+## 📌 Hook State {#state-hook}
 
-This example renders a counter. When you click the button, it increments the value:
+Contoh berikut me-*render* sebuah counter. Jika tombol di klik, nilai akan bertambah:
```

Comment:
Italicize  *counter* if you want to keep it in English, otherwise you can translate the example code to **penghitung**.